app/src/main/java/com/example/testjavaapp/data/DataHolder.java

```diff
@@ -0,0 +1,17 @@
+package com.example.testjavaapp.data;
+
+import android.graphics.Bitmap;
+
+import java.util.ArrayList;
+
+public class DataHolder {
+
+    private ArrayList<Bitmap> data;
+
+    public ArrayList<Bitmap> getBitmaps() {return data;}
```

Comment:
Inconsistency between class name and method name

Reply:
Fixed